pkgs/applications/science/logic/isabelle/default.nix

```diff
@@ -71,7 +77,7 @@ in stdenv.mkDerivation (finalAttrs: rec {
 
   sourceRoot = "${dirname}${lib.optionalString stdenv.isDarwin ".app"}";
 
-  doCheck = true;
+  doCheck = stdenv.hostPlatform.system != "aarch64-linux";
```

Comment:
If it builds and runs on Linux/arm64, why don't the tests work?

Reply:
The check phase just hangs. I don't know why, but I've seen [this](https://mailmanbroy.informatik.tu-muenchen.de/pipermail/isabelle-dev/2021-November/017554.html) which says that not all the proofs in HOL-SMT_Examples work on arm64-linux.